### PR TITLE
feat(docs): added anchor links in Landing page in products section

### PR DESCRIPTION
**Pull request by Shanugoyanka**

### CODE REVIEW

File: components/products.tsx [hunk 1]
```diff
@@ -56,25 +58,31 @@ const Section: React.FC<SectionProps> = ({ heading, items }) => {
...
+    route: '/guides',
+    list:[
     { title: 'Hosting', description: (<>The Agentverse hosting platform enables all users to get started quickly and to deploy agents to the cloud to start connecting and automating.</>),icon: walletIcon , path: "/concepts/agent-services/agent-hosting"},
     { title: 'Mailbox', description: (<>Set up mailboxes for your local agents and to run them independently of your constant presence to run the server.</>),icon: mailBoxIcon , path: "/concepts/agent-services/agent-mail"},
     { title: 'Agent APIs', description: (<>Understand and use the Agentverse APIs.</>),icon: authenticationIcon , path: "/apis/agentverse"},
     { title: 'Explorer', description: (<>Learn to use the Agentverse Explorer to start an interaction with other registered agents.</>),icon: walletIcon , path: "/concepts/agent-services/agent-explorer"},
-  ],
-  'Open Network': [
+  ]},
+  'Open Network': {
```
Comment (devjsc) [inline components/products.tsx]: Does this work? 

if i go to /docs#opennetwork will i land at this section?

Comment (Shanugoyanka) [reply components/products.tsx]: No Josh, it won't work, I have added routes to this header

Comment (devjsc) [reply components/products.tsx]: Okay, but that is the feature that is being requested - let's sync on this

Comment (Shanugoyanka) [reply components/products.tsx]: I have also added this functionality

**Pull request merged**